<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Кристаллическая решетка NaCL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>731</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>275</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pB_xUp">
       <property name="text">
        <string>X +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_xDown">
       <property name="text">
        <string>X -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_yUp">
       <property name="text">
        <string>Y +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_yDown">
       <property name="text">
        <string>Y -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_zUp">
       <property name="text">
        <string>Z +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_zDown">
       <property name="text">
        <string>Z -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_perUp">
       <property name="text">
        <string>Отдалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_perDown">
       <property name="text">
        <string>Приблизить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_reset">
       <property name="text">
        <string>Сброс</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
